dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -13,11 +13,15 @@
 # limitations under the License.
 
 import tensorflow as tf
+from tensorflow.python.data.util import nest
```

Comment:
Why `nest`?

Reply:
To handle the same cases as TF that uses this.